### PR TITLE
base.html styles

### PR DESCRIPTION
**Pull request by YarinBou**
this is a new design for our navbar and resposivity.
73771828-aeda-411c-b1e6-2378f5b6ebcf
98866b08-367d-4cd8-80b4-498a9cede606
r

### CODE REVIEW

File: SJMaster/templates/base.html [hunk 1]
```diff
@@ -1,45 +1,41 @@
...
-            rel="stylesheet"
-            integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
-            crossorigin="anonymous"
-    />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <!-- google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@100;200;400&display=swap" rel="stylesheet">
```
Comment (guyy13) [inline SJMaster/templates/base.html]: Can you change it to this link so we have all the weights:
`<link href="https://fonts.googleapis.com/css2?family=Raleway:wght@100;200;300;400;600;700&display=swap" rel="stylesheet">`

Comment (YarinBou) [reply SJMaster/templates/base.html]: yes

Comment (YarinBou) [reply SJMaster/templates/base.html]: done.

**Review by guyy13 — APPROVED** [review]

**Review by eldaraharonson — APPROVED** [review]

**Review by benmenachem — APPROVED** [review]

**Review by yaelerdr — APPROVED** [review]

File: SJMaster/settings.py [hunk 2]
```diff
@@ -129,6 +129,11 @@
 ]
 LOGIN_REDIRECT_URL = "/"
 LOGOUT_REDIRECT_URL = "/"
+STATICFILES_DIRS = [
```
Comment (guyy13) [inline SJMaster/settings.py]: Please remove this as it's already in the file. (was added on #82 ).

**Review by guyy13 — CHANGES_REQUESTED** [review]

File: SJMaster/settings.py (hunk 2 above)
Comment (YarinBou) [reply SJMaster/settings.py]: done

**Review by guyy13 — APPROVED** [review]

**Comment by guy9050** [comment]
squash the commits ("resolve conflicts")

**Comment by YarinBou** [comment]
@guy9050 done.

**Review by guy9050 — APPROVED** [review]
lgtm

**Pull request merged**